source/2019-10-18-the-ember-times-issue-120.md

```diff
@@ -34,13 +34,15 @@ READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [A New EmberMap Series: What's New in Ember 📚](https://embermap.com/topics/what-s-new-in-ember/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+This year, Ember released many new features that will all be a part of Ember Octane. While the [release notes](https://blog.emberjs.com/tags/releases.html) do a great job of summarizing the features, you may be still wondering about what they are and why they matter to you as a developer or an addon contributor.
 
```

Comment:
I feel that this part of `you may be still wondering about what they are and why they matter` is not really clear and a long sentence. Could you see if there is a way to make it shorter or split it up in multiple sentences?

Reply:
Yeah, I agree that it was a long, confusing sentence. Can you check the revision to see if it makes more senes? Thanks.